### ISSUE TITLE
Unable to import specific class in Python Input code

### ISSUE THREAD

**Issue by Noobzik** [amphi-ai/amphi-etl#187]
Used with Minio
```bash
python -m pip install minio
```
Step to reproduce : 
1. Drag and drop 'Python input'
2. Click on the config icon
3. Paste these code to the import section
```python
import pandas as pd
import minio
import sys
import os
import platform
```
4. Paste these line of code to the second section
```python
env_info = {
    "Python Executable": sys.executable,
    "Python Version": platform.python_version(),
    "Minio version": minio.__version__
 }

output = pd.DataFrame([env_info])
```

This should show the current path of the python executable and his version as a pandas DataFrame
<img width="1253" alt="image" src="https://github.com/user-attachments/assets/32db9120-2386-41c4-bbd5-ddd72616baf4">
5.  Now add this to imports:
```python
from minio import Minio
```
6. And add the following code
```python
client = Minio(
        "localhost:9000",
        secure=False,
        access_key="minio",
        secret_key="minio123"
    )
```

<img width="586" alt="image" src="https://github.com/user-attachments/assets/3e3ec964-7ab4-487b-8e88-262110163571">

**Comment by tgourdel** [comment]
Thanks for reporting @Noobzik, will fix today.

**Comment by tgourdel** [comment]
Should be fixed in 0.8.8! Thanks
